catalyst/core/logger.py

```diff
@@ -22,6 +22,15 @@ def __init__(self, log_batch_metrics: bool, log_epoch_metrics: bool) -> None:
         self._log_batch_metrics = log_batch_metrics
         self._log_epoch_metrics = log_epoch_metrics
 
+    @property
+    def logger(self) -> Any:
+        """Internal logger/experiment/etc. from the monitoring system.
+
+        Returns:
+            Any: internal logger/experiment/etc. from the monitoring system.
+        """
+        return None
```

Comment:
🤔 could we make it abstract & raise NotImplementedError for those, where it can not be done? like csv, or so
I think it would be much better, if we would try to guarantee the internal logger support :)
but while it's `None` by default... it could be missed from the implementation.